5_databases/week1/homework

```diff
@@ -0,0 +1,55 @@
+-- Active: 1699216915341@@0.0.0.0@3306@hyf_lesson1
+
+SELECT COUNT(*)
+ FROM task;
+
+
+
+
+SELECT COUNT(*) 
+FROM task 
+WHERE due_date <= CURDATE();
+
+
+
+SELECT *
+FROM task
+WHERE status_id= 3;
+
+
+SELECT *
+FROM task
+WHERE status_id!= 3;
+
+
+
+SELECT *
+FROM task
+ORDER BY created DESC;
+
+
+SELECT MAX(created)
+from task;
+
+SELECT title, due_date
+FROM task;
+
+
+SELECT task.title, status.name
```

Comment:
This returns the task 3 times with all statuses which means 103 rows of tasks, but we only have 35 tasks. We only want 1 task and its current status:

A little extra help for this:
SELECT task.title, status.name
FROM task
JOIN status
ON status.id=task.status_id;